plugins/plugins.pro

```diff
@@ -14,12 +14,14 @@ AUTHENTICATION_PLUGINS = \
     inappauthplugin
 
 CRYPTO_PLUGINS = \
-    opensslcryptoplugin
+    opensslcryptoplugin \
+    cryptokiplugin
 
 SUBDIRS+=\
     $$STORAGE_PLUGINS \
     $$ENCRYPTION_PLUGINS \
     $$ENCRYPTEDSTORAGE_PLUGINS \
     $$AUTHENTICATION_PLUGINS \
     $$CRYPTO_PLUGINS \
+    $$CRYPTOKI_PLUGINS \
```

Comment:
this isn't needed, I guess, since the cryptoki plugin is listed under CRYPTO_PLUGINS already.